scipy/optimize/_linprog.py

```diff
@@ -65,13 +65,17 @@ def linprog_verbose_callback(res):
             feasible solution is sought and the T has an additional row
             representing an alternate objective function.
         status : int
-            An integer representing the exit status of the optimization::
+            An integer representing the exit status of the optimization:
 
-                 0 : Optimization terminated successfully
-                 1 : Iteration limit reached
-                 2 : Problem appears to be infeasible
-                 3 : Problem appears to be unbounded
-                 4 : Serious numerical difficulties encountered
+            ``0`` : Optimization terminated successfully
```

Comment:
Trying to be consistent at least within `linprog`: these exist status lists have no bullets, and the number is in teletype.

Not attempting to fix other sections of optimize here. Some use teletype numbers, some use bullets, etc.